caffe2/contrib/aten/gen_op.py

```diff
@@ -92,7 +92,10 @@ def value_is_tensor_type(v):
     'std::array<bool, 2>': 'auto ${arg} = readBoolMask<2>("${arg}");',
     'std::array<bool, 3>': 'auto ${arg} = readBoolMask<3>("${arg}");',
 }
-
+for _k in list(ARGUMENT_MAP):
+    __k = _k.replace(' ','')
+    if __k not in ARGUMENT_MAP:
+        ARGUMENT_MAP[__k] = ARGUMENT_MAP[_k]
```

Comment:
What's going on here?

Reply:
This fixes the following warning when building pytorch:
```
Skipping conv_dilated2d_backward Because of Arg: std::array<bool,3> (std::array<bool,3>)
```
Note that the original `ARGUMENT_MAP` contains a key `std::array<bool, 3>` (with space after comma) and this change adds a space-less variant of the key to `ARGUMENT_MAP`. The space-less variant is what is actually used when using `ARGUMENT_MAP`.

Reply:
Oh I see. Actually, probably the spaced variant is not necessary, and we should only have the unspaced variant.

Reply:
ok, I'll remove the for loop and remove the spaces from `ARGUMENT_MAP` keys.